swarm/kubernetes.go

```diff
@@ -95,10 +118,12 @@ func NewClientKubernetes(kubernetesInCluster bool, kubernetesURL string) (*Clien
 		httpClient: httpClient,
 		apiURL:     apiURL,
 		token:      token,
+		retry:      backoff.NewConstantBackOff(5 * time.Second),
```

Comment:
i think we should use the backoff.WithMaxRetries because otherwise the backoff.Retry will never return in case of permanent failure